DataFormats/Common/interface/MergeableDouble.h

```diff
@@ -0,0 +1,14 @@
+#ifndef Common_MergeableDouble_h
+#define Common_MergeableDouble_h
+
+namespace edm {
+
+  struct MergeableDouble {
```

Comment:
better make a proper class, also include a default constructor with initialization of value